### DISCUSSION TITLE
Does Unikraft support the functions of sys/timerfd

### DISCUSSION THREAD

**Discussion by chenzq-98** [unikraft/unikraft#765] [General]
Does Unikraft support the functions of sys/timerfd? I select the systemcall_shim but compiling result shows that "sys/timerfd.h: No such file or directory"

**Comment by razvand** [comment]
@czq693497091, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
